openapi/src/resources/azure_vm.yaml

```diff
@@ -21,6 +21,10 @@ components:
           description: A valid WSM identifier for an Azure Disk that corresponds to a valid azure resource
           type: string
           format: uuid
+        priority:
```

Comment:
I figured these 2 things made sense to be in the Azure VM response, instead of just creation parameters.